Example/Example/Tests/Helpers/Float.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Float.swift
+//  Example
+//
+//  Created by Mathias Claassen on 2/13/18.
+//
+
+import Foundation
+
+extension Array where Element == Float {
+
+    static func random(count: Int) -> [Float] {
+        var arr = [Float]()
+        for _ in 0..<count {
+            arr.append(Float(arc4random()) / Float(UINT32_MAX))
+        }
+        return arr
```

Comment:
what about `return (0..<count).map { _ in Float(arc4random()) / Float(UINT32_MAX) }` ?